android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -23,18 +23,52 @@
 
     private String      adUnitId;
     private MaxAdFormat adFormat;
-
-    private String  placement;
-    private String  customData;
-    private Boolean adaptiveBannerEnabled;
-    private Boolean autoRefresh;
+    private String      placement;
+    private String      customData;
+    private Boolean     adaptiveBannerEnabled;
```

Comment:
nit but booleans should be preceded by "is" (this results in linter warning, and is generally accepted programming convention)

Reply:
Yes, but these are the properties that are set by Javascript via the View Manager, so I think that the names are better to be the same as the Javascript properties.